Amsterdam-Music-Lab/MUSCLE
# Added: Add 2x3 grid support for Matching Pairs component if it has equal or less than 6 sections

Related to #652 

## How to test this?

Creating an MatchingPairs-like experiment with six or less sections should result into the `MatchingPairs` component to be displayed with only two columns.

A quicker route may be to look a the component in Storybook. ~You can do this by running the following command: `docker compose run --rm client yarn run storybook`, although it (connecting to Storybook running inside of a Docker container) does not seem to work on my pc (anymore?) at the time of writing this...~ 😓

**Update:**

The correct Storybook command is `docker compose run --rm -p 6006:6006 client yarn run storybook` (which opens the correct port).
